crypto/ml_dsa/ml_dsa_sign.c

```diff
@@ -43,12 +43,110 @@ static void signature_init(ML_DSA_SIG *sig,
 }
 
 /*
- * FIPS 204, Algorithm 7, ML-DSA.Sign_internal()
- * @returns 1 on success and 0 on failure.
+ * @brief: Auxiliar functions to compute ML-DSA's MU.
```

Comment:
Auxiliary ?

Reply:
Please suggest any wording that works for you.

Reply:
There is just a typo auxiliar -> auxiliary

Reply:
aahh, sorry I am slightly dislexic, sometimes I do not see those typos at all :)